src/Microsoft.Security.Utilities.Core/WellKnownRegexPatterns.cs

```diff
@@ -92,6 +92,15 @@ public static IEnumerable<RegexPattern> HighConfidenceSecurityModelsIterator()
         new AzureEventGridIdentifiableKey(),
     };
 
+    public static IEnumerable<RegexPattern> DataClassification { get; } = new RegexPattern[]
+    {
+        new FormalGuid(),
```

Comment:
>FormalGuid [](http://example.com/codeflow?start=12&length=10)

this order is neither alphabetical (one possible consistent ordering standard) nor waterfalled by length (a second ordering standard).

If you take all my notes, here's how I would have organized these checks:

        new IPv4(),
        new IPv6(),
        new Guid(),
        new Uuid(),
        new Float(),
        new Integer(),

Reply:
You didn't create this technical debt but generally we should be encoding the readable name every time we refer to a friendly name. In this case, devs can just f12 into the thing though....